Makefile

```diff
@@ -111,7 +111,7 @@ build_rosetta_all_sigs: ocaml_checks
 
 build_intgtest: ocaml_checks
 	$(info Starting Build)
-	dune build --profile=integration_tests src/app/test_executive/test_executive.exe src/app/logproc/logproc.exe
+	dune build --profile=$(DUNE_PROFILE) src/app/test_executive/test_executive.exe src/app/logproc/logproc.exe
```

Comment:
Can we at least default this to use `integratin_tests` when no dune profile is provided? The integration tests won't execute correctly with the default dev profile, for instance.